### PR TITLE
redirect to the upcoming debate if there is one, or the last debate if not

### PR DESCRIPTION
**Pull request by tobiasmcnulty**

### CODE REVIEW

File: opendebates/tests/test_views.py [hunk 1]
```diff
@@ -17,6 +17,43 @@
...
+            debate_time=timezone.now() + datetime.timedelta(days=20),
+        )
+        r = self.client.get('/')
+        self.assertRedirects(r, '/%s/' % self.next_debate.prefix)
+
+    def test_only_previous_debate(self):
+        self.next_debate.delete()
+        r = self.client.get('/')
+        self.assertRedirects(r, '/%s/' % self.previous_debate.prefix)
+
```
Comment (dchukhin) [inline opendebates/tests/test_views.py:58]: Could you also add a test for the case that there is no `Debate` for the relevant `Site`?

**Review by dchukhin — COMMENTED** [review]
I like this behavior much better

File: opendebates/tests/test_views.py (hunk 1 above)
Comment (tobiasmcnulty) [reply opendebates/tests/test_views.py:58]: Good idea! Added.

**Review by dchukhin — APPROVED** [review]
▶️

**Pull request merged**